sunbeam-python/sunbeam/core/deployment.py

```diff
@@ -332,7 +332,7 @@ def parse_manifest(self, manifest_data: dict) -> Manifest:
 
     def get_manifest(self, manifest_file: pathlib.Path | None = None) -> Manifest:
         """Return the manifest for the deployment."""
-        if self._manifest is not None:
+        if self._manifest is not None and not manifest_file:
```

Comment:
I would not expect this feature to need to change core behaviour of the manifest.

Can you please give any sense of why this is needed or if this was just a workaround that should be removed?

Reply:
At the time, `self._manifest` would have been already been set when enabling the `baremetal` feature, meaning that the given `manifest_file` would have been ignored (manifest file given as: `sunbeam enable --manifest ./some-manifest.yaml baremetal`). That doesn't seem to be the case anymore, so I'm fine with removing this.